src/lib/y2storage/clients/finish.rb

```diff
@@ -0,0 +1,67 @@
+# encoding: utf-8
+
+# Copyright (c) [2018] SUSE LLC
+#
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of version 2 of the GNU General Public License as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE LLC about this file by physical or electronic mail, you may
+# find current contact information at www.suse.com.
+
+require "yast"
+require "installation/finish_client"
+
+module Y2Storage
+  module Clients
+    # Finish client for storage-related tasks
+    class Finish < ::Installation::FinishClient
+      # Constructor
+      def initialize
+        textdomain "storage"
+        Yast.import "Service"
```

Comment:
NP: just matter of taste, but Yast.import is almost same as `require` so can be together unless there is reason to import it only in constructor

Reply:
I checked it and most finish clients do it that way. So I decided to stay consistent.